doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2275,6 +2275,15 @@ CIA-351: Connectivity issues with :ref:`lib_azure_iot_hub`
 Other libraries
 ===============
 
+.. rst-class:: v2-4-2 v2-4-1 v2-4-0
+
+NCSDK-23315: The :ref:`bt_le_adv_prov_readme` has improper range and default value of the :kconfig:option:`CONFIG_BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE` Kconfig option.
+  The buffer size has not been aligned to new Fast Pair not discoverable advertising data size after size of salt included in the data was increased from 1 byte to 2 bytes.
+  As a consequence of it, the :ref:`peripheral_fast_pair` sample reboots on assert when it has five Account Keys and starts advertising in the Fast Pair not discoverable advertising mode.
+  This happens because the buffer is too small to fit the Fast Pair not discoverable advertising data when five Account Keys are present.
+
+  **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``a8b668e82837295962348e9e681125c2ae11bb4e``).
+
```

Comment:
We need to have a new section "Bluetooth libraries and services" and this KI could be moved to that section as this library don't belong to "other libraries".